Tensile/TensileCreateLibrary.py

```diff
@@ -1506,13 +1528,19 @@ def splitExtraParameters(par):
         masterFile = os.path.join(archPath, "TensileLibrary")
         newMasterLibrary.applyNaming(kernelMinNaming)
         LibraryIO.write(masterFile, Utils.state(newMasterLibrary), args.LibraryFormat)
+  elif globalParameters["SeparateArchitectures"]:
+    for archName, newMasterLibrary in masterLibraries.items():
+      if (archName in archs):
```

Comment:
Don't need parentheses here.

Reply:
Thanks for catching that, I fixed that and removed a commented out line